src/tools/r2rdump/R2RDump.csproj

```diff
@@ -1,5 +1,5 @@
-﻿<Project Sdk="Microsoft.NET.Sdk" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-<Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
```

Comment:
Can we add a condition on ArcadeBuild=="" on the import?

Reply:
I don't understand why in this case. It was present before and is needed regardless of Arcade or not.